cloud/ociutil/ociutil.go

```diff
@@ -116,7 +112,11 @@ func AwaitLBWorkRequest(ctx context.Context, loadBalancerClient lb.LoadBalancerC
 			wr = &twr.WorkRequest
 			return true, nil
 		case loadbalancer.WorkRequestLifecycleStateFailed:
-			return FetchErrorsOnFailedWorkRequest(ctx, workRequestClient, workRequestId)
+			final_err := errors.Errorf("WorkRequest %s failed", *workRequestId)
```

Comment:
Why is there no call to `networkLoadBalancerClient.ListWorkRequestErrors` here?

Reply:
Loadbalancer provides the respective work-request-error details in the getWorkRequest call itself, unlike nlb which has a separate api.